good-citizen.md

```diff
@@ -0,0 +1,33 @@
+# Being a Good Citizen of Exercism 
+
+We want Exercism to be a great place to spend time, whoever you are.
+To ensure that Exercism is that, we place a strong focus on being good community citizens.
+This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**
+
+## The words we use
+
+There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.
+
+Words in themselves are meaningless. 
+They are vibrations caused by puffs of air coming out of our lungs.
+However, because of **context**, words are powerful.
+
+It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
```

Comment:
```suggestion
It's easy to know how you intend _your_ words to come across, but it's difficult to know how others _hear your words_.
```

Reply:
```suggestion
It's easy to know how you intend your words to _come across_, but it's difficult to know how others _hear your words_.
```

Reply:
Thoughts on my version @BethanyG - I tried pairing the two phrases slightly differently. See which you think is clearer :)

Reply:
I like the second one.  😄   Better emphasis -- and better example of the problem.